src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -186,6 +193,11 @@ def _get_add_headers_callback(cli_ctx):
 
     def _add_headers(request):
         agents = [request.headers['User-Agent'], UA_AGENT]
+        try:
+            agents.append(os.environ[ENV_AZ_INSTALLER])
```

Comment:
The current UA is like 

```
'User-Agent': 'python/3.8.2 (Windows-10-10.0.18362-SP0) msrest/0.6.11 msrest_azure/0.6.2 azure-mgmt-resource/8.0.1 Azure-SDK-For-Python AZURECLI/2.2.0'
```

Maybe we can append the content of `ENV_AZ_INSTALLER` to `UA_AGENT` and make it like `AZURECLI/2.2.0 (PIP)`?

https://github.com/Azure/azure-cli/blob/31a9724478c67751ae0bee6cc0d9b75b763df17c/src/azure-cli-core/azure/cli/core/commands/client_factory.py#L18

This is like what Chrome does: 

```
Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Mobile Safari/537.36
```

My two cents.